record-builder-core/src/main/java/io/soabase/recordbuilder/core/RecordBuilder.java

```diff
@@ -1,12 +1,12 @@
 /**
  * Copyright 2019 Jordan Zimmerman
- *
+ * <br>
```

Comment:
This is coming from the license file and is not hand coded. This is the standard ALS license. I think this is not correct.

Reply:
Makes sense. I have reverted the line break change. Though the IDE still suggests adding a line break tag or a paragraph tag. Otherwise adding the line break doesn't makes any affect as it won't be rendered as a line break in Javadoc.

Reply:
To avoid such case, I open https://github.com/Randgalt/record-builder/pull/145 and you can give it a review :)